article/app/model/dotcomponents/DotcomponentsDataModel.scala

```diff
@@ -513,7 +503,6 @@ object DotcomponentsDataModel {
     implicit val DotComponentsDataModelWrites = new Writes[DotcomponentsDataModel] {
       def writes(model: DotcomponentsDataModel) = Json.obj(
         "page" -> model.page,
-        "page2" -> model.page2,
```

Comment:
Well done getting rid of that :)

Reply:
@shtukas unfortunately I didn't solve the problem, as soon as we try and add 1 more property to `DCPage` we'll run into the same problem again! 😭